api-catalog-services/src/main/java/com/ca/mfaas/apicatalog/instance/InstanceRefreshService.java

```diff
@@ -42,11 +45,17 @@
 
     /**
      * Periodically refresh the container/service caches
+     * Depends on the GatewayClient: no refreshes happen when it's not initialized
```

Comment:
I'd rather explain what will happen if "it's initiated". For example "The GatewayClient initialization refreshes the container/service cache"